pkgs/applications/window-managers/yabar/default.nix

```diff
@@ -1,38 +1,8 @@
-{ stdenv, fetchFromGitHub, cairo, gdk_pixbuf, libconfig, pango, pkgconfig, xcbutilwm }:
+{ callPackage, ... }@args:
 
-stdenv.mkDerivation rec {
-  name    = "yabar-${version}";
+callPackage ./build.nix {
   version = "0.4.0";
 
-  src = fetchFromGitHub {
-    owner  = "geommer";
-    repo   = "yabar";
-    rev    = "746387f0112f9b7aa2e2e27b3d69cb2892d8c63b";
-    sha256 = "1nw9dar1caqln5fr0dqk7dg6naazbpfwwzxwlkxz42shsc3w30a6";
-  };
-
-  buildInputs = [ cairo gdk_pixbuf libconfig pango pkgconfig xcbutilwm ];
-
-  hardeningDisable = [ "format" ];
-
-  postPatch = ''
-    substituteInPlace ./Makefile --replace "\$(shell git describe)" "${version}"
-  '';
-
-  buildPhase = ''
-    make DESTDIR=$out PREFIX=/
-  '';
-
-  installPhase = ''
-    make DESTDIR=$out PREFIX=/ install
-    mkdir -p $out/share/yabar/examples
-    cp -v examples/*.config $out/share/yabar/examples
-  '';
-
-  meta = with stdenv.lib; {
-    description = "A modern and lightweight status bar for X window managers";
-    homepage    = "https://github.com/geommer/yabar";
-    license     = licenses.mit;
-    platforms   = platforms.linux;
-  };
-}
+  rev = "d3934344ba27f5bdf122bf74daacee6d49284dab";
```

Comment:
Are you sure you want the same `rev` for 0.4.0 and unstable?

Reply:
I think you want `rev = version;`.

Reply:
Sorry, just noticed you had changed it. I still think it is a good idea to use `rev = version;` to avoid future mistakes :-)